float32[] body_pos
float32[] body_rpy
float32[] body_linear_vel
float32[] body_angular_vel
float32[] joint_pos
float32[] joint_vel
